sys/trace/Kconfig.config

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+# XXX This file only is required since there is no easy way to use these config
+# options with both the final MODULE_SHELL and KCONFIG_USEMODULE_SHELL in a
+# nicely looking and easy to migrate way. After migration, the content of this
+# file can be folded back into `sys/shell/Kconfig`
```

Comment:
Copy-pasta. Should say trace, not shell :-)

Reply:
Nice one, I made it more copy-pasta-able.